### PR TITLE
Replace nbsp by new class sidebar-menu-item

### PR DESCRIPTION
**Pull request by yubiuser**
- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

**What does this PR aim to accomplish?:**

Replace nbsp by new class sidebar-menu-item in sidebar menu

### CODE REVIEW

File: style/pi-hole.css [hunk 1]
```diff
@@ -330,6 +330,10 @@
   background: #367fa9;
 }
 
+.sidebar-tab {
```
Comment (dschaper) [inline style/pi-hole.css]: Would `sidebar-menu-item` or `sidebar-menuitem` fit in closer to what the element is?

Comment (yubiuser) [reply style/pi-hole.css]: I am not particularly attached to a certain name....

Comment (dschaper) [reply style/pi-hole.css]: Okay, to me a `sidebar-tab` would be for the entire section and this is for styling the actual menu item. So a `sidebar-menu-item` makes sense and could be expanded to `sidebar-menu-subitem`.

I haven't checked but the updated AdminLTE may have addressed this already? We are using an old version of the template.

**Review by Relluem94 — APPROVED** [review]

**Review by DL6ER — COMMENTED** [review]
Where are the `</span>` ?

**Comment by yubiuser** [comment]
I left them out after reading [here](https://www.freecodecamp.org/news/tab-space-instead-of-multiple-non-breaking-spaces/), that they are self-closing and can be omitted. Do you want me to include them?

> I haven't checked but the updated AdminLTE may have addressed this already? We are using an old version of the template.

I think they have addressed this. They have a `nav-item` with `nav-link` which adds a `nav-icon` https://adminlte.io/docs/3.0/components/main-sidebar.html

**Comment by DL6ER** [comment]
> after reading here, that they are self-closing and can be omitted

Browsers will only interpret all self closing tags correctly if the mime type is some of XHTML or XML. I don't think this works for `text/html` mime-type.

---

I just looked at how the spacing actually happens and it appears to me that something like
```
.fa-fw{
  margin-right: 5px;
}
```
adding to `pihole.css` alone would do the trick - no changes to `headers.php` required.

**Comment by DL6ER** [comment]
Or, to be explicit about the sidebar:
``` css
.sidebar-menu .fa-fw {
  margin-right: 5px;
}
```

**edit** Replaced `section.sidebar .fa-fw` by `.sidebar-menu .fa-fw`

This adds the margin highlighted in yellow here:
Screenshot from 2021-05-05 18-35-23

File: style/pi-hole.css [hunk 2]
```diff
@@ -330,6 +330,10 @@
   background: #367fa9;
 }
 
+.sidebar-menu-item {
+  padding-left: 5px;
+}
+
```
Comment (DL6ER) [inline style/pi-hole.css:336]:
```suggestion
.sidebar-menu .fa-fw {
  margin-right: 5px;
}
```

and remove all changes from `header.php`

**Comment by yubiuser** [comment]
In-vivo development disturbed the branches. Follow up PR https://github.com/pi-hole/AdminLTE/pull/1799

**Pull request closed without merging**